mpc-recovery/src/oauth.rs

```diff
@@ -57,8 +57,17 @@ fn validate_jwt(
         ..
     } = &claims;
 
-    if !oidc_providers.contains(issuer, audience) {
-        anyhow::bail!("UnauthorizedTokenIssuerOrAudience: iss={issuer}, aud={audience}");
+    match oidc_providers {
+        Some(oidc_providers) => {
+            if !oidc_providers.contains(issuer, audience) {
+                anyhow::bail!("UnauthorizedTokenIssuerOrAudience: iss={issuer}, aud={audience}");
+            }
+        }
+        None => {
+            // If no OIDC providers are specified in the allowlist, we allow any issuer and audience.
+            // Should be used in signing nodes only.
+            tracing::warn!("No OIDC providers found in the allowlist");
```

Comment:
this will produce a lot of noise in the logs, since this will be ran on every sign. We should have it at node initialization instead